_includes/master/charts/calico/values.yaml

```diff
@@ -1,4 +1,39 @@
-typha:
-  enabled: false
+# Select which datastore mode. Can be 'etcd' or 'kdd'.
+# Note: Changing this after installation is not supported. 
+datastore: kdd
+
+# Sets the networking mode. Can be 'calico', 'flannel', or 'none'
 network: "none"
+
+# Global docker image registry. This will set the image registry for all images *except* flannel.
+imageRegistry: "calico/"
+
 calico_ipam: false
+app_layer_policy: false
+
+node:
+  image: calico/node
+  tag: master
+calicoctl:
+  image: calico/ctl
+  tag: master
+typha:
+  image: calico/typha
+  tag: master
+cni:
+  image: calico/cni
+  tag: master
+kubeControllers:
+  image: calico/kube-controllers
+  tag: master
+flannel:
+  image: quay.io/coreos/flannel
+  tag: v0.9.1
+dikastes:
+  image: calico/dikastes
+  tag: master
+flexvol:
+  image: calico/flexvol
+  tag: master
+prodname: Calico
```

Comment:
Probably worth documenting these two values down here as well.

Reply:
I left these two undocumented because I want to remove them somehow.

Reply:
Hm, how can we remove these? Where are they used right now?